cookbook-react-native/recipes/images/loadable-image/readme.md

```diff
@@ -0,0 +1,50 @@
+# Cookbook React Native Bootstrap: LoadableImage Recipe
+
+![recipe](https://raw.githubusercontent.com/Wolox/frontend-cookbook/master/cookbook-react-native/recipes/images/loadable-image/recipe.gif)
+
+## Installation
+
+`yarn add react-native-fast-image`
+
+or
+
+`npm install react-native-fast-image`
+
+and
+
+`npx pod-install`
```

Comment:
:que-trucazo:

Reply:
yes, and you don't need to move to`/ios` to run this 😄